File: Front-End/lib/enums/payment_status_enum.dart
enum PaymentStatusEnum {
  inBasket,    // 0
  registered,  // 1
  inWaiting,   // 2
  cancelled,   // 3
}

PaymentStatusEnum paymentStatusFromInt(int value) {
  return PaymentStatusEnum.values[value];
}

int paymentStatusToInt(PaymentStatusEnum status) {
  return status.index;
}

File: API/LocalDBApi/LocalDBApi/Enums/PaymentStatusEnum.cs
namespace LocalDBApi.Enums
{
    public enum PaymentStatusEnum
    {
        None = 0,
        InBasket = 1,
        Registered = 2,
        InWaiting = 3,
        Cancelled = 4
    }
    public static class PaymentStatusEnumExtensions
    {
        public static string ToText(this PaymentStatusEnum status) => status switch
        {
            PaymentStatusEnum.Registered => "ثبت نام شده",
            PaymentStatusEnum.InBasket => "در انتظار پرداخت",
            PaymentStatusEnum.InWaiting => "در لیست انتظار شما",
            PaymentStatusEnum.Cancelled => "کنسل شده",
            _ => "نامشخص"
        };
    }
}



